### ISSUE TITLE
VerificationException: Operation could destabilize the runtime

### ISSUE THREAD

**Issue by ghost** [accord-net/framework#134]
Hi Guys,

I'm a real fan of the framework and love what you're doing with it. I do seem to have an issue integrating it into an ASP.NET site. 

When using it, I get an error message "Operation could destabilize the runtime" with a System.Security.VerificationException. I have read up online and it seems that the issue is related to code security inside the Accord.NET binaries. 

Could anyone enlighten me if there is anything I can do with regards to solving this issue?

![untitled](https://cloud.githubusercontent.com/assets/14232650/9811750/67866238-5879-11e5-9308-bf8123bc9990.png)

**Comment by cesarsouza** [comment]
Hello there,

Of course, sorry for the very late reply. At the time it was also not clear for me, but it seems this could had been result of an issue on the .NET framework itself (which I believe was version 4.5 at the time) and which was probably fixed in this update: https://support.microsoft.com/en-us/help/2748645/an-update-is-available-for-the-.net-framework-4.5-in-windows-7,-windows-7-sp1,-windows-server-2008-r2,-windows-server-2008-r2-sp1,-windows-server-2008-sp2,-and-windows-vista-sp2

However, it is very likely that this issue will be solved completely once #318 is finished and the framework can officially support .NET Core and thus ASP.NET Core.

**Comment by cesarsouza** [comment]
Since #318 was finished, it is very likely that this issue should have been addressed as well. If anyone still encounters this issue while working with the most recent version of the framework, please re-open or create a new issue here in the issue tracker referencing this one.
